sources/packages/api/src/services/education-program/education-program.service.ts

```diff
@@ -135,7 +145,10 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
     program.intlExchangeProgramEligibility =
       educationProgram.intlExchangeProgramEligibility;
     program.programDeclaration = educationProgram.programDeclaration;
-
+    program.statusUpdatedBy = { id: educationProgram.userId } as User;
+    if (createProgram) {
+      program.submittedBy = { id: educationProgram.userId } as User;
```

Comment:
I didn't get this part. If educationProgram.id is present it is create otherwise, it must be update. Am I missing something?

Reply:
yes, you are correct